### DISCUSSION TITLE
Tutorial 2: MultiApps has an invalid link

### DISCUSSION THREAD

**Discussion by doquang** [idaholab/moose#18425] [Q&A Tools] [answered]
- The page (https://mooseframework.inl.gov/moose/getting_started/examples_and_tutorials/tutorial02_multiapps/index.html) has an invalid link: https://mooseframework.org/getting_started/examples_and_tutorials/tutorial02_multiapps/slides
- The page (https://mooseframework.inl.gov/syntax/Transfers/index.html) has an invalid link to the MultiApps: https://mooseframework.inl.gov/MultiApps/index.md. I think the correct link is: https://mooseframework.inl.gov/syntax/MultiApps/index.html

**Comment by GiudGiud** [comment, marked as answer]
Thank you for reporting this!

There's a PR up for the website to fix the first point momentarily. 
In the mean time, you can build this documentation locally with
```
cd tutorials/tutorial02_multiapps/doc
python moosedocs.py build --serve
```

I ll make a PR to address the second point.
